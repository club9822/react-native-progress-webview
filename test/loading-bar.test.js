import React from 'react';
import {shallow} from 'enzyme';
import LoadingBar from '../lib/loading-bar';

describe('LoadingBar', () => {
	describe('Rendering', () => {
		it('should match to snapshot', () => {
			const component = shallow(<LoadingBar />);
			expect(component).toMatchSnapshot()
		});
	});
});
