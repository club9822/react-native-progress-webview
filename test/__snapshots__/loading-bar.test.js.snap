// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBar Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
